### ISSUE TITLE
Double-check solar radiation limit

### ISSUE THREAD

**Issue by stasinos** [data-eng/navgreen#44]
Solar radiation values are limited to 2 (cf. navgreen_base/processing.py, l.56), and useful values are above 0.15 (cf. recent email thread).

But the value histogram in the csv is:

(-1000, 0]         347732
(0, 2]               6172
(2, 10]             15011
(10, 20]            14650
(20, 30]            12370
(30, 40]            11446
(40, 50]             9303
(50, 100]           27764
(100, 200]          42595
(200, 300]          33898
(300, 400]          26599
(400, 500]          23168
(500, 600]          23121
(600, 700]          23853
(700, 800]          26015
(800, 900]          30961
(900, 1000]         40417
(1000, 1000000]     17152

Given these values, restricting between 0.15 and 2 seems odd. Can this be a unit conversion bug?

**Comment by tatiana-boura** [comment]
Yes, actually, the live data is converted to the 'useful' units. 
I will fix it now, by adjusting the `navgreen_base/preprocessing.py` script accordingly.

**Comment by stasinos** [comment]
Maybe its better to sync the record with the way live data is written, to avoid unnecessary branching of the preprocessing logic.
Are the new historical data written in kWatts? Then we just need to convert the old csv file.

**Comment by tatiana-boura** [comment]
In the preprocessing file I have already (prior to this Issue) made the distinction between whether we process live or hist data, with hist data being the data acquired prior to our involvement in the project, so adjusting those data is just a minor addition to the current code. 

The data historical data after our involvement is in the converted units (in this case kWatts, yes)

**Comment by stasinos** [comment]
That makes sense. Cross-referencing issue #45 here just so that the note is recorded.
